www/source/partials/docs/_dev-pkgs-binding.html.md.erb

```diff
@@ -44,7 +44,7 @@ pkg_binds=(
 )
 ```
 
-Here, `pkg_binds` is an associative array. The key (here, "database") is the binding name, while the value (here, "port ssl-port") is a space-delimited list of the exported configuration the binding requires. A consumer can have specify multiple binds; each would be an individual entry in this associative array. Judging from this, the producer we described above would be a good candidate for this application server to bind to, because it exports both a "port" and an "ssl-port".
+Here, `pkg_binds` is an associative array. The key ("database") is the binding name, while the value ("port ssl-port") is a space-delimited list of the exported configuration the binding requires. A consumer can have specify multiple binds; each would be an individual entry in this associative array. Judging from this, the producer we described above would be a good candidate for this application server to bind to, because it exports both a "port" and an "ssl-port".
```

Comment:
I'll add these in a commit, rather than using the UI.

Reply:
Feel free to always incorporate the feedback in the way that works best for you. I like using the "suggested change" interface since it makes it easy to see the exact difference between the PRs change and the reviewer suggestion.